canvas_modules/common-canvas/src/common-properties/components/control-item/control-item.jsx

```diff
@@ -46,12 +46,15 @@ class ControlItem extends React.Component {
 			let tooltip;
 			if (this.props.control.description && !isEmpty(this.props.control.description.text)) {
 				if (this.props.control.description.placement === "on_panel") {
-					description = <div className="properties-control-description">{this.props.control.description.text}</div>;
+					description = <div className="properties-control-description">{this.props.control.description.text}</div>; // add link here???
 				// only show tooltip when control enabled and visible
 				} else {
 					tooltip = (<Tooltip
 						id={`${this.uuid}-tooltip-label-${this.props.control.name}`}
 						tip={this.props.control.description.text}
+						link={this.props.control.description.link ? this.props.control.description.link : null}
+						tooltipLinkHandler={this.props.controller.getHandlers().tooltipLinkHandler}
+						propertyId={this.props.propertyId} // is there a better way to handle this w/o passing propertyId to the tooltip??
```

Comment:
As tooltip is a shared component between canvas and properties, how to avoid passing `propertyId` to the Tooltip component? We need propertyId when calling `tooltipLinkHandler` callback.

Reply:
That's a good point.  We can probably leave off propertyId and only use the `id`